hotness/consumers.py

```diff
@@ -284,8 +284,7 @@ def _handle_anitya_update(self, upstream, package, msg):
                                 else:
                                     self.new_triggered_task_ids[task_id] = [bz, None, str(upstream), str(package)]
                 else:
-                    note = 'Patching or scratch build for %s and version %s FAILED.\n' \
-                           'See for details' % (package, version)
+                    note = 'Patching or scratch build for %s and version %s FAILED.\n' % (package, version)
```

Comment:
Sorry but to me this English sentence is really difficult to understand. I will only like to suggest drop "and version" words from above. Also, we in general always write or say 
"<name>-<version>" together and not separately.